### PR TITLE
virTemplate.md submission for final project

### PR DESCRIPTION
**Pull request by jheeswyk**

### CODE REVIEW

File: template/img/alignment.PNG
Comment (ababaian) [inline template/img/alignment.PNG:1]: please place all images in a specific folder for your Obelisk and not in the `template` folder but in the `virus` folder. See other submissions from the last year for examples of how filepaths should look like. Amend the paths in the Markdown submission as well so it works.

File: template/img/alignment.PNG:Zone.Identifier
Comment (ababaian) [inline template/img/alignment.PNG:Zone.Identifier:1]: remove empty file.

File: template/virTemplate.md
Comment (ababaian) [inline template/virTemplate.md:1]: Please don't modify the `virTemplate.md` file directly in your PR. Instead copy the file and re-name it to match your submission. See how the files are arranged in other submission.s

File: template/virTemplate.md [hunk 1]
```diff
@@ -1,40 +1,140 @@
...
+## Sequence comparison of Clostridiae-associata centralis with experimental sequencing data from BioProject 
+
+To investigate whether Clostridiae-associata centralis is present in the Clostridium beijerinckii genome, I performed a tBLASTn search [NCBI BLAST] with my query sequence as ORF2 from Clostridiae-associata centralis and my chosen search set as the experimental data associated with BioProject PRJNA315559 (ERX2023530). My goal was to see if the putative Oblin-1 ORF in Clostridiae-associata centralis was present in the BioProject sequencing data, which would provide insight as to whether functional sequence from Clostridiae-associata centralis could be found in Clostridium beijerinckii. The result returned over 100 alignments with percent identity over 85% and E values under 0.01 (see Fig. 2).
+
+![Fig. 2: Results from tBLASTn with Clostridiae-associata centralis ORF2 query and ERX2023530 chosen search set.](img/alignment.png)
+
 
-{Q2 answer ...}
 
 ### Virus Genome {Q3}
```
Comment (ababaian) [inline template/virTemplate.md]: Amend out the "Default Titles" from the document. Your descriptive titles are much better throughout.

File: template/virTemplate.md [hunk 2]
```diff
@@ -1,40 +1,140 @@
...
 
-{Insert your Virus Genome Map Figure and figure legend here}
+![Fig. 1A: ORFs of Clostridiae-associata centralis as determined by ORFfinder, with nucleotide scale bar. Created with R package gggenes based on GitHub PuravG/Computational-Virus-Discovery](img/genome.png)
```
Comment (ababaian) [inline template/virTemplate.md]: if the "Figure Legend" is in the title, it won't be displayed, The square-bracket text here is the "title" shown if the image does not load. Instead put the text in italics underneath the figure.

**Review by ababaian — CHANGES_REQUESTED** [review]
Submission looks really cool :) I'm looking forward to diving into it. Please amend the PR to fix the formatting requirements.

**Comment by jheeswyk** [comment]
Hi Artem,
I edited my submission file and renamed it to Clostridiae-associata_centralis.md, changed the filepaths for it and my images as requested, and reverted virTemplate.md to the unfilled version. Please let me know if everything looks good!

File: template/.Rhistory
Comment (ababaian) [inline template/.Rhistory:1]: Please remove/delete dotfiles from the repo.

**Review by ababaian — APPROVED** [review]

**Pull request merged**